app/views/cases/offender_sar/_case_details.html.slim

```diff
@@ -8,31 +8,31 @@
     table
       tbody.sar-basic-details
 
-        = render partial: 'cases/shared/case_type', locals: { case_details: case_details }
+        = render partial: 'cases/shared/case_type', locals: { case_details: case_details, can_be_edit: can_be_edit }
```

Comment:
Nit picking, sorry - I wonder if we could rename `can_be_edit` to `allow_editing` or something like that?